pkg/api/types.go

```diff
@@ -1190,6 +1190,8 @@ type ServiceSpec struct {
 	// Type determines how the service will be exposed.  Valid options: ClusterIP, NodePort, LoadBalancer
 	Type ServiceType `json:"type,omitempty"`
 
+	LoadBalancerIP string `json:"loadBalancerIP, omitEmpty`
```

Comment:
you probably want this to be LoadBalancerIngress like in status. The pattern here is usually request something in spec, have it allocated in status, so it'll be weird if they differ.